src/elements/Header/Header.jsx

```diff
@@ -10,7 +10,7 @@ export default {
     dividing: Boolean,
     disabled: Boolean,
     floated: Enum(['left', 'right']),
-    icon: Boolean,
```

Comment:
You should still be able to render an "icon" header and specify where to show the icon (it could be after the text).

I'd expect a user to pass a string only if there is no other content in the header (see the [react version](https://react.semantic-ui.com/elements/header))

eg:

```html
<sui-header icon>
   content
  <sui-icon name="settings" />
</sui-header>
```

vs

```html
<sui-header icon="settings" />
```